hardware/arduino/avr/libraries/HID/HID.cpp

```diff
@@ -86,10 +86,13 @@ void HID_::AppendDescriptor(HIDSubDescriptor *node)
 	descriptorSize += node->length;
 }
 
-void HID_::SendReport(uint8_t id, const void* data, int len)
+int HID_::SendReport(uint8_t id, const void* data, int len)
 {
-	USB_Send(pluggedEndpoint, &id, 1);
-	USB_Send(pluggedEndpoint | TRANSFER_RELEASE, data, len);
+    auto ret = USB_Send(pluggedEndpoint, &id, 1);
+    if(ret >= 0){
+        ret += USB_Send(pluggedEndpoint | TRANSFER_RELEASE, data, len);
```

Comment:
Yes but 0 would also indicate an error? Well I dont really know whats best here since you always have this dilemma whit different return types across differen functions. But I think size_t would fit better.